BBD Review: Sidebar, definitions, indexing, and related cleanup (#503)
worldbank/dime-data-handbook: chapters/4-acquisition.tex

  DIME guidelines for encryption in SurveyCTO/ODK are part of the DIME Standards:
  \url{https://github.com/worldbank/dime-standards/}}
If your data collection service allows you to browse data in the browser,
then the encryption is only implemented correctly if you are asked for the key each time.

The data security standards that apply
when receiving confidential data from the field
also apply when transferring confidential data to the field,
such as sampling or enumeration lists containing PII.\index{data transfer}
In some survey software,
you can use the same encryption that allows you to receive data securely
from the field, to also send confidential data,
such as an identifying list of respondents, to the field.
Otherwise, you will need to create a securely stored file,
transfer it to the field team using an insecure tool,
and have them decrypt the information locally
using a key that is transferred using a secure password manager.
This process will be more similar to that for securely \textit{storing} data,
which we discuss next.

% doesn't cover 'big' data
The first thing you need to do before planning how to securely send or receive data,

> is to plan how to securely store data after you have received it.

Reviewer: ```suggestion
is to plan how to securely store data after it has been transferred.
```
Since we now say "you send or receive" we cannot say "you have received"
